.github/workflows/gradle-data-capturing-samples-verification.yml

```diff
@@ -42,3 +42,10 @@ jobs:
       - name: Run Gradle build
         working-directory: common-gradle-enterprise-gradle-configuration
         run: ./gradlew tasks
+      - name: Inject data capture Kotlin script into Gradle build
+        run: |
+          # apply sample file
+          echo "apply from: file(\"../build-data-capturing-gradle-samples/${{matrix.sample-file}}.kts\")" >> common-gradle-enterprise-gradle-configuration/build.gradle
```

Comment:
I'd rather not name the kotlin sample .`gradle.kts` and have the extension added depending on the GH action step.

Reply:
How would you organize the Kotlin files ? Currently they are sitting next to their groovy counterpart with same name, different extension (.kts)

Reply:
My impression is that a different extension is enough to differentiate the content. I don't think that a separate hierarchy would really help.

Reply:
ah then I'm not sure to understand your initial comment, could you clarify ?

Reply:
I would name the samples `foo.kts` and not `foo.gradle.kts`

Reply:
Ah, it looks like the current convention is to name Gradle Kotlin scripts `*.gradle.kts`

Reply:
my bad then, you can dismiss this comment